Examples/zib-Infusion-01.xml

```diff
@@ -1,74 +1,9 @@
 <DeviceUseStatement xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/deviceusestatement.xsd">
+<!--Disclaimer: This is a minimal example of Infusion (DeviceUseStatement). Some references may contain another information (like patient). -->
```

Comment:
-Disclaimer: This is a minimal example of Infusion (DeviceUseStatement). Some references may contain another information (like patient)

Miss is dit beter?

-Disclaimer: This is a minimal example of Infusion (DeviceUseStatement) and may not be realistic. Some references may contain incorrect and not matching information.